runtime/gc_modron_startup/mmparseXgc.cpp

```diff
@@ -585,6 +585,10 @@ j9gc_initialize_parse_gc_colon(J9JavaVM *javaVM, char **scan_start)
 		extensions->dnssAvoidMovingObjects = true;
 		goto _exit;
 	}
+	if(try_scan(scan_start, "enableScanConstantPoolObjectInScavenge")) {
+		extensions->enableScanConstantPoolObjectInScavenge = true;
+		goto _exit;
+	}
```

Comment:
Do we need this option in final change? Do we have any plans to do more performance runs?